simplq/src/components/pages/Admin/index.jsx

```diff
@@ -25,14 +25,20 @@ export default () => {
 
   const [items, setItems] = useState();
 
-  const update = () => {
+  const update = (updateClicked) => {
     if (queueId) {
       QueueService.get(queueId)
         .then((data) => {
           setItems(data.tokens);
+          if (!updateClicked) {
```

Comment:
Using updateClicked to determine if this is a call from polling or from a button click might be messy. Shall I just create a separate function pollForUpdates instead of using if's everywhere?

Reply:
Yes, please. It'll be cleaner.